src/System.Drawing.Common/src/System/Drawing/Bitmap.cs

```diff
@@ -195,239 +110,123 @@ public Bitmap(Stream stream, bool useIcm)
-        /// class, from the specified existing image, with the specified size.
-        /// </devdoc>
         public Bitmap(Image original, int width, int height) : this(width, height)
         {
-            Graphics g = null;
-            try
+            using (Graphics g = Graphics.FromImage(this))
             {
-                g = Graphics.FromImage(this);
                 g.Clear(Color.Transparent);
                 g.DrawImage(original, 0, 0, width, height);
             }
-            finally
-            {
-                if (g != null)
-                {
-                    g.Dispose();
-                }
-            }
         }
 
-        /// <include file='doc\Bitmap.uex' path='docs/doc[@for="Bitmap.FromHicon"]/*' />
-        /// <devdoc>
-        ///    Creates a <see cref='System.Drawing.Bitmap'/> from a Windows handle to an
-        ///    Icon.
-        /// </devdoc>
         public static Bitmap FromHicon(IntPtr hicon)
         {
             IntPtr bitmap = IntPtr.Zero;
-
             int status = SafeNativeMethods.Gdip.GdipCreateBitmapFromHICON(new HandleRef(null, hicon), out bitmap);
+            SafeNativeMethods.Gdip.CheckStatus(status);
 
-            if (status != SafeNativeMethods.Gdip.Ok)
-                throw SafeNativeMethods.Gdip.StatusException(status);
-
-            return Bitmap.FromGDIplus(bitmap);
+            return FromGDIplus(bitmap);
         }
 
-        /// <include file='doc\Bitmap.uex' path='docs/doc[@for="Bitmap.FromResource"]/*' />
-        /// <devdoc>
-        /// </devdoc>
         public static Bitmap FromResource(IntPtr hinstance, String bitmapName)
         {
             IntPtr bitmap;
-
             IntPtr name = Marshal.StringToHGlobalUni(bitmapName);
 
             int status = SafeNativeMethods.Gdip.GdipCreateBitmapFromResource(new HandleRef(null, hinstance),
                                                               new HandleRef(null, name),
                                                               out bitmap);
             Marshal.FreeHGlobal(name);
+            SafeNativeMethods.Gdip.CheckStatus(status);
 
-            if (status != SafeNativeMethods.Gdip.Ok)
-            {
-                throw SafeNativeMethods.Gdip.StatusException(status);
-            }
-
-            return Bitmap.FromGDIplus(bitmap);
+            return FromGDIplus(bitmap);
         }
 
-        /// <include file='doc\Bitmap.uex' path='docs/doc[@for="Bitmap.GetHbitmap"]/*' />
-        /// <devdoc>
-        ///    <para>
-        ///       Creates a Win32 HBITMAP out of the image. You are responsible for
-        ///       de-allocating the HBITMAP with Windows.DeleteObject(handle). If the image uses
-        ///       transparency, the background will be filled with the specified background
-        ///       color.
-        ///    </para>
-        /// </devdoc>
         [SecurityPermission(SecurityAction.LinkDemand, Flags = SecurityPermissionFlag.UnmanagedCode)]
```

Comment:
Need not be in this change, but all the [SecurityPermission] stuff can be deleted.